_episodes/06-good-chart.md

```diff
@@ -10,10 +10,48 @@ keypoints:
 - "A good chart directly and clearly addresses a scientific question." 
 ---
 
-Introduce [Choosing a Good Chart][good-chart].
+## Simple steps to choosing a good chart
 
-Break down chart decision for: "How was the population of Africa distributed
-across the continent by country in 2007?"
+1. Identify/define the research question you wish to address.
+2. Determine the broad category that describes the goal of your research
+   question from [Choosing a Good Chart][good-chart]. 
+   - *You may need to redefine
+   or focus your research question to better suit visualization.*
```

Comment:
I think you're right that conceptually the note is part of the main bullet. I just don't like how it looks. It extends the line across the page (unlike the other points) and has an awkward wrap. If you feel strongly about it, I would be happy to make the change or consider a different option.